app/src/main/java/com/example/baseandroid/utils/StringUtils.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.baseandroid.utils
+
+import java.util.*
+
+object StringUtils {
+    private fun isNullOrEmpty(text: String?): Boolean {
```

Comment:
**nitpick:** `isNullOrEmpty` method is already available for Kotlin Collections framework. We can remove this method.